### ISSUE TITLE
Units of property functions

### ISSUE THREAD

**Issue by larschrj** [ClapeyronThermo/Clapeyron.jl#250]
It would be nice if the documentation explicitly stated that properties are per unit mol when dealing with a single fluid (for the functions that are not already explicitly intensive such as density). That might be the assumption for some people but certainly not all 😉. 

Also, there seems to be a copy-paste error in the docstring for mass_density. It states that it calculates the molar density, not mass density.

**Comment by longemen3000** [comment]
hi!,
on documenting that, do you mean as a preface? all functions that are volume derivatives tend to be intensive (pressure does not depend on the amount of matter), but temperature derivative functions (like Cp and Cv) are extensive. each function has their units documented, but adding additional information never hurts.

on the doc error, i'm fixing that right away

**Comment by larschrj** [comment]
Some comment in the preface would be helpful at least to me but I take your point that the information is there in the documentation 🙂.

As a tangent to this, it would be nice if mass could passed as an argument to f.ex. volume when dealing with single fluids like IAPWS95 (which is what I have been using so far). I noticed in the docs that this is an option when dealing with mixtures. It would save some tedious calculations for me, but perhaps it is not a good idea for some reason.

**Comment by larschrj** [comment]
My mistake. I must have made an error when trying this previously. That's great 🙂 Thanks 👍

**Comment by longemen3000** [comment]
I will keep this open in the meantime. The typo in mass_density was corrected. There is a docs PR that should fix this, but it isn't merged yet.

**Comment by larschrj** [comment]
Although if I pass the vector argument for the mass I get a no method matching error for a_res. But in any case your solution was what I was looking for.

**Comment by longemen3000** [comment]
Yeah, `a_res` is more of an internal function 😅, but there should be an api to access those (something like `helmholtz_free_energy_res` that I thought we had)

**Comment by larschrj** [comment]
Oo

**Comment by ianhbell** [comment]
Aside: it should be ``helmholtz_energy_res`` rather than ``helmholtz_free_energy_res`` according to IUPAC

**Comment by runjaj** [comment]
> single fluids are mixtures! (of only one component), you can pass a mass argument in the following way:
> 
> ```
> julia> volume(model,1u"atm",30u"°C",1u"kg",phase = :l, output = u"L")
> 1.0043695560170631 L
> ```

I have tried to do this using the last version of Clapeyron.jl and I get an error:

```julia
julia> using Clapeyron, Unitful

julia> model = IAPWS95()
MultiParameter Equation of state for water:
 Polynomial power terms: 7
 Exponential terms: 44
 Gaussian bell-shaped terms: 3
 Non Analytic terms: 2

julia> volume(model,1u"atm",30u"°C",1u"kg",phase = :l, output = u"L")
ERROR: FieldError: type SingleFluid has no field `params`, available fields: `components`, `properties`, `ancillaries`, `ideal`, `residual`, `references`
Stacktrace:
 [1] getproperty
   @ ./Base_compiler.jl:54 [inlined]
 [2] mw(model::SingleFluid{EmpiricAncillary})
   @ ClapeyronUnitfulExt ~/.julia/packages/Clapeyron/OWffP/ext/ClapeyronUnitfulExt.jl:34
 [3] standardize(model::SingleFluid{…}, x::Quantity{…}, st::Quantity{…})
   @ ClapeyronUnitfulExt ~/.julia/packages/Clapeyron/OWffP/ext/ClapeyronUnitfulExt.jl:51
 [4] standardize(model::SingleFluid{…}, x::Quantity{…}, T::Quantity{…}, z::Quantity{…})
   @ ClapeyronUnitfulExt ~/.julia/packages/Clapeyron/OWffP/ext/ClapeyronUnitfulExt.jl:26
 [5] volume(model::SingleFluid{…}, p::Quantity{…}, T::Quantity{…}, z::Quantity{…}; phase::Symbol, threaded::Bool, vol0::Nothing, output::Unitful.FreeUnits{…})
   @ ClapeyronUnitfulExt ~/.julia/packages/Clapeyron/OWffP/ext/ClapeyronUnitfulExt.jl:140
 [6] top-level scope
   @ REPL[4]:1
Some type information was truncated. Use `show(err)` to see complete types.

julia> using Pkg

julia> Pkg.status("Clapeyron")
Status `~/OneDrive/Documents/FEA/apuntes-fea/Project.toml`
  [7c7805af] Clapeyron v0.6.16
```

It works if I use the molecular weight of water:

```julia
julia> volume(model,1u"atm",30u"°C",1u"kg"*1u"mol"/18u"g",phase = :l, output = u"L")
1.0052214845937855 L
```

Is this an expected behavior or am I doing anything wrong?

**Comment by longemen3000** [comment]
Hi, this is a bug, let me look at it
